### PR TITLE
Dladowitz/improve file uploader button

### PR DESCRIPTION
**Pull request by dladowitz**

### CODE REVIEW

**Comment by mockdeep** [comment]
needs a rebase

File: app/views/listings/_form.html.haml [hunk 1]
```diff
@@ -80,5 +83,11 @@
   }
 
   $("#file-uploader").change(function() {
+    console.log("file-uploader")
+    readURL(this);
+  });
+
+  $("#customFile").change(function() {
+    console.log("customFile")
```
Comment (mockdeep) [inline app/views/listings/_form.html.haml]: Can we remove these `console.log`s?

File: app/views/listings/_form.html.haml [hunk 2]
```diff
@@ -37,18 +37,21 @@
...
+    #preview-images-container.mt-2.mb-2
 
     %hr
     .input-group.mb-5
       = label_tag "Add Categories"
       = select_tag "categories[]", options_for_select(@categories),{ class: "ui fluid search dropdown", id: "categories", :multiple => "", :prompt => 'Search' }
-      #preview-images-container.mt-2
+
+
+
```
Comment (mockdeep) [inline app/views/listings/_form.html.haml]: Some extra newlines here.

File: app/views/listings/_form.html.haml (hunk 1 above)
Comment (dladowitz) [reply app/views/listings/_form.html.haml]: done

File: app/views/listings/_form.html.haml (hunk 2 above)
Comment (dladowitz) [reply app/views/listings/_form.html.haml]: and done

**Pull request merged**